packages/autocomplete/src/index.ts

```diff
@@ -139,8 +154,10 @@ async function completer(params: AutocompleteParameters, line: string): Promise<
       let expressions;
       if (splitLine[2].match(/\baggregate\b/)) {
         // aggregation needs extra accumulators to autocomplete properly
-        expressions = BASE_COMPLETIONS.concat(getStageAccumulators(
-          params, elToComplete));
+        expressions = ([] as AnyCompletions).concat(
+          BASE_COMPLETIONS,
+          ...getStageAccumulators(params, elToComplete)
```

Comment:
Curious, is there a reason to spread here instead of what we were doing before?

Reply:
Oh, actually the spread here is not really required, yeah, I might've added it just out of muscle memory, but not for any good reason. I did change concatenating from being done on `BASE_COMPLETIONS` to empty array because I find that asserting type makes more sense when reading code that way, but tell me if it doesn't make sense to you, I'll switch it back

Reply:
All good, makes sense, was wondering if there was a reason for doing it that might not be apparent.